data/mrabro.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Rafi Abro",
+  "type": "personal",
+  "bio": "I'm Rafi, a full-time full-stack developer 👨‍💻 working since 2016 🚀",
+  "avatar": "https://github.com/mrabro.png",
+  "tags": ["Open Source", "wordpress", "javascript", "nodejs", "DevOps"],
+  "socials": [
+      { "platform": "twitter", "url": "https://twitter.com/mrabro" },
+      { "platform": "github", "url": "https://github.com/mrabro" },
+      { "platform": "youtube", "url": "https://youtube.com/mrabro" }
```

Comment:
Please check & update your YouTube URL with the correct one. It seems to be broken! 🛠

![image](https://user-images.githubusercontent.com/88786642/206105470-4f9b24af-cc1c-4d9b-b8a4-686d25dde3fe.png)

Reply:
oh, I forgot to replaced the platform and url: its actually facebook :)